group/templates/group/profile.html

```diff
@@ -1,6 +1,47 @@
 {% extends "main/layout.html" %}
 
 {% block body %}
-    <h1>You created Group. Good job !</h1>
-    {{form_data}}
+    {% if group.cover_photo %}
+        <img src="{{ group.cover_photo.url }}" alt="group photo">
+    {% endif %}
+
+    <h1>{{ group.name }}</h1>
+    <p>Details: {{ group.description }}</p>
+    <p>Location: {{ group.location }}</p>
+    <p>Visibility: {{ group.visibility }}</p>
+    <p>Join Mode: {{ group.join_mode }}</p>
+    <p>Capacity: {{ group.capacity }}</p>
+    <p>Created At: {{ group.created_at }}</p>
+    <p>Created By: {{ group.creator }}</p>
+    <p> Admins: 
+        {% for admin in group.admins.all %}
+            {{ admin.username }},
+        {% endfor %}
+    </p>
+    <p>Members: {{ group.members.all.count }}</p>
+    <p> Interests: 
+        {% for interest in group.interests.all %}
+            {{ interest.name }},
+        {% endfor %}
+    </p>
+    <div class="d-flex justify-content-center">
+        {% if user.is_authenticated %}
+            {% if user in group.admins.all %}
+                <a class="btn btn-secondary m-2" href="#">Create Event</a>
```

Comment:
I think Our behavior for this "Create Event" we want them to see this button. so that there is an option that group members can create event but requires admin permission. so encouraging for them to work for group.